minseo/app.js

```diff
@@ -36,6 +36,53 @@ app.get("/ping", function (req, res, next) {
   res.json({ message: "pong" });
 });
 
+app.post("/posts", async (req, res) => {
+  const { title, content, userId } = req.body;
+
+  await appDataSource.query(
+    `INSERT INTO posts(
+    title,
+    content,
+    user_id
+  )VALUES(?, ?, ?);`,
+    [title, content, userId]
+  );
+  res.status(201).json({ message: "postCreated" });
+});
+
+app.get("/posts", async (req, res) => {
+  await appDataSource.query(
+    `SELECT
+      users.id AS userId,
+      posts.id AS postingId,
+      posts.content AS postingContent
+      FROM users
+      JOIN posts ON users.id = posts.user_id
+      `,
+    (err, rows) => {
+      res.status(200).json(rows);
+    }
+  );
+});
+app.get("/user_posts", async (req, res) => {
```

Comment:
```suggestion
app.get("/users/posts", async (req, res) => {
```
http target, 즉 end-point에는 가급적 under bar를 쓰지 않도록 지양해주세요. 계층 관계가 있을 때는 `/`를 활용해주시면 됩니다. 자세한 내용은 앞으로 있을 RESTful API 세션에서 보다 더 상세하게 다룰 예정이나, 궁금하신점이 있으시면 찾아와 질문해주세요!